src/scripts/main.js

```diff
@@ -1,3 +1,28 @@
 'use strict';
 
-// write your code here
+const tableData = document.querySelectorAll('td');
+
+const selectorsToRender = ['thead > tr', 'tfoot > tr'];
+
+selectorsToRender.forEach(selector => {
+  const thToInsert = document.createElement('th');
+
+  thToInsert.innerText = 'Position';
+
+  document.querySelector(selector).children[3].after(thToInsert);
+});
```

Comment:
Let's use `row` property of `table` and `cloneNode` method

Reply:
Thanks for hint. I didn't suppose that `table `has `rows `prop. This property shorted my solution.